design-system/BottomSheet/BottomSheetModal.tsx

```diff
@@ -40,8 +43,15 @@ export const BottomSheetModal = memo(
           containerComponent={
             Platform.OS === "ios" ? renderContainerComponent : undefined
           }
+          enableDynamicSizing={false} // By default we don't want enable dynamic sizing
           backdropComponent={BottomSheetBackdropOpacity}
           handleComponent={renderHandleComponent}
+          backgroundStyle={[
+            {
+              backgroundColor: theme.colors.background.raised,
+            },
+            backgroundStyle,
+          ]}
```

Comment:
I've put your change at the design-system level because I think that's the styling we'll want to use for all our bottom sheet anyway